compiler/rustc_trait_selection/src/traits/select/mod.rs

```diff
@@ -293,6 +316,18 @@ impl<'cx, 'tcx> SelectionContext<'cx, 'tcx> {
         self.infcx.tcx
     }
 
+    /// returns `true` if the predicate is considered `const` to
```

Comment:
```suggestion
    /// Returns `true` if the predicate is considered `const` to
```

Reply:
I will change this after this lands or on this PR if CI fails